.github/ISSUE_TEMPLATE/bug-template.yaml

```diff
@@ -25,7 +25,7 @@ body:
         - Workers for Platforms
         - Constellation
```

Comment:
We should probably rename this to Workers AI

Reply:
```suggestion
        - Workers AI
```